<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdNewTextDialog</class>
 <widget class="QDialog" name="EdNewTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>186</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type In New Text for Node Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNewText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
